frontend/assets/styles/sass/global.sass

```diff
@@ -7,7 +7,7 @@ body,
 
 html,
 body,
-    font-family: 'Lato', sans-serif
+    font-family: 'BCSans', sans-serif
```

Comment:
By adding the importation of font to the parent element, you dont have to declare the font throughout the child elements. So we were able to save a ton of lines.